fonts.css

```diff
@@ -0,0 +1,22 @@
+@font-face {
+	font-family: 'MuseoSansCyrl';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/baliproject/pullRequest?prid=5558708)